addons/website_blog/static/src/snippets/s_latest_posts/000.js

```diff
@@ -48,10 +58,10 @@ publicWidget.registry.js_get_posts = publicWidget.Widget.extend({
                         class: 'alert alert-warning alert-dismissible text-center',
                         text: _t("No blog post was found. Make sure your posts are published."),
                     })));
-                    return;
```

Comment:
@sdegueldre This, alongside with the option not working that you mentioned in your commit should be backported once I finish merging it.